### DISCUSSION TITLE
Same functionality - but for tabs?

### DISCUSSION THREAD

**Discussion by codeIcannot** [Cookie-AutoDelete/Cookie-AutoDelete#1077] [Ideas / Feature Requests]
Hello. I've been using Cookie-AutoDelete for some time now, and I really like how it works. But I'm also drowning in excess tabs! So I'm curious if it would be possible to create an extension, that "auto-deletes" (closes) tabs? I'm a bit of a tab-hoarder and it would be nice if they would automatically close, after a certain, predefined period. Like 14 days, or whatever. I'd also like to mark some tabs as "never close these" and "hold for now". The extension would then list how many tabs are protected, or on hold. If that makes sense?

Anyway. Thank you for the great CAD extension!

**Comment by kennethtran93** [comment]
I'm also a tab-hoarder myself, with firefox close to 100 tabs in various containers.  This is an idea, though to be honest one I would not use myself.

**Reply by kennethtran93** [reply]
Though in the case of 'undo' it may not be as easy.
